Resources/Prototypes/DeltaV/GameRules/events.yml

```
@@ -207,3 +207,45 @@
         prototype: Recruiter
       - type: RoleBriefing
         briefing: Find candidates, conduct interviews and seal the deal by having them sign with your special recruiter's pen.
+
+- type: entity
+  parent: BaseGameRule
+  id: SynthesisSpecialist
+  components:
+  - type: StationEvent
+    weight: 4
+    minimumPlayers: 20
+    maxOccurrences: 1
+    duration: null
```

```suggestion
    duration: 1 
```
Same effect, but no reason to make it null